app/forms/additional_claimants_upload_form.rb

```diff
@@ -15,7 +15,7 @@ class AdditionalClaimantsUploadForm < Form
     form.validates :additional_claimants_csv, content_type: {
       in: %w<text/csv text/plain>,
       message: I18n.t('errors.messages.csv')
-    }
+    }, unless: :remove_additional_claimants_csv
```

Comment:
The content validator won't work on the second run because the file isn't in the cache(local) & so calling path on the file object won't work but we can skip this if we are removing the file.